src/main/java/io/tiledb/java/api/Query.java

```diff
@@ -1121,6 +1110,190 @@ public Object getBuffer(String attr) throws TileDBError {
     return (long[]) buffer.toJavaArray(nelements);
   }
 
+  /**
+   * Retrieves an IntBuffer of an attribute attr of type Integer
+   *
+   * @param attr The attribute name
```

Comment:
Let's change this to `buffer` instead of `attr`/`attribute` so there is no confusion about this works for dimensions too. Let's do this for all the functions.